calcculator-app/src/components/Input.js

```diff
@@ -0,0 +1,277 @@
+import React from "react";
+import Grid from "./Grid.js";
+import Mod from "./Mod.js";
+import Equal from "./Equal.js";
+import Clear from "./Clear.js";
+import "./styles/Input.css";
+
+class Input extends React.Component {
+  state = {
+    input: [0],
+    firstDisplay: [],
+    mod: "",
+    secondDisplay: [],
+    answer: 0,
+    // equal:
+  };
+
+  handleInputChange = (event) => {
+    //let display = event.target.value.split("");
+  
+    if (this.state.mod) {
+      const { name, value } = event.target; //this makes the state update with each change. Not quite sure why. It was part of forms lesson. I think its deconstructing in some way.
+      this.setState({ [name]: value }); // this is part of that^^
+      console.log(this.state);
+      this.setState((prevState) => ({
+        secondDisplay: parseInt([...prevState.input]), //parseInt removes zero
+      }));
+      console.log(this.state);
+      //debugger;
+    } else {
+      const { name, value } = event.target; //this makes the state update with each change. Not quite sure why. It was part of forms lesson. I think its deconstructing in some way.
+      this.setState({ [name]: value }); // this is part of that^^
+      console.log(this.state);
+      this.setState((prevState) => ({
+        firstDisplay: parseInt([...prevState.input]), //parseInt removes zero
+      }));
+    }
+  };
+
+  handleGrid = (event) => {
+    let num = 56789345667
+    console.log(num.toLocaleString())
+    console.log(this.state);
```

Comment:
What are we trying to accomplish with our `num` variable on 41?  If this is a test of `toLocaleString()` we should remember to remove it before pushing.